lib/pushmi_pullyu/cli.rb

```diff
@@ -180,14 +180,14 @@ def run_preservation_cycle
           # Log successful preservation event to the log files
           PushmiPullyu::Logging.log_preservation_event(deposited_file)
         end
-      # rubocop:disable RescueWithoutErrorClass
+      # rubocop:disable Style/RescueStandardError
```

Comment:
Not sure what to do about this: rubocop likes this, hound doesn't. I thought it might be a version mismatch issue of rubocop, but that doesn't seem to be the case.

Reply:
Whew, seem to have found the issue.